astropy/coordinates/tests/test_finite_difference_velocities.py

```diff
@@ -210,6 +210,30 @@ def test_altaz_diffs():
     assert np.all(np.sum(cdiff.d_xyz**2, axis=0) ** 0.5 > constants.c)
 
 
+def test_dt_function():
+    gcrs_coord = GCRS(
```

Comment:
This test shows that nothing breaks, but not really that the finite difference is used. I think it would be easy to show it is by adding a second set (same function OK in this case) with a large dt (half a year should give drastically bad answers...), and then check that the answer differs.

Reply:
I've parametrized the test.